### ISSUE TITLE
/opt/ros/noetic/lib/rviz/rviz: Missing libtinyxml2.so.9

### ISSUE THREAD

**Issue by julianboehm** [arch4edu/arch4edu#269]
/opt/ros/noetic/lib/rviz/rviz: error while loading shared libraries: libtinyxml2.so.9: cannot open shared object file: No such file or directory

For lack of a better idea, I tried symlinking /usr/lib/libtinyxml2.so (10.0.0-1) which ros-noetic-rviz depends on, resulting in:
rviz: error while loading shared libraries: libpython3.11.so.1.0: cannot open shared object file: No such file or directory
and, after trying to symlink libpython3.12.so.1.0:
rviz: error while loading shared libraries: libglog.so.1: cannot open shared object file: No such file or directory
where the latest installed version (0.7.0-1) is not compatible (missing symbol).

Do the ros-noetic-* packages need to be rebuilt with the latest libraries?

### Steps to reproduce
1. Install Arch Linux
2. Add arch4edu repository to `pacman.conf`
3. `pacman -Syu && pacman -S ros-noetic-desktop-full`
4. `source /opt/ros/noetic/setup.bash`
5. `rviz`

**Comment by petronny** [comment]
> Do the ros-noetic-* packages need to be rebuilt with the latest libraries?

Yes they do but it's still in progress. According to https://arch4edu.org/ , `ros-noetic-rviz` is waiting for `ros-noetic-std-msgs` to be rebuilt. But currently the build `ros-noetic-std-msgs` is failing.

**Comment by cybaol** [comment]
A PR is waiting for merging: https://github.com/ros-noetic-arch/ros-noetic-catkin/pull/8

**Comment by cybaol** [comment]
@petronny https://github.com/arch4edu/cactus/actions/runs/9492323982/job/26159271998

**Comment by petronny** [comment]
47d5843316748da4dba1715452aa608cc0f42b26

**Comment by petronny** [comment]
#308
